README.md

```diff
@@ -450,7 +450,7 @@ Currently, the following report formats are supported (reporter names are case-i
 # System requirements
 
 ORT is being continuously used on Linux, Windows and macOS by the
-[core development team](https://github.com/orgs/oss-review-toolkit/teams/core-devs), so these operating systems are
+[core development team](https://github.com/orgs/oss-review-toolkit/people), so these operating systems are
```

Comment:
Commit message "README: ..."

Also, strictly speaking the link is not dead, but just inaccessible for non-organization-members (which is quite stupid by GitHub to do it this way).

Reply:
Fixed the README part and reworded it from `fix dead link` to `fix inaccessible link`.